go.mod

```diff
@@ -197,7 +196,6 @@ require (
 replace (
 	github.com/go-logr/logr => github.com/go-logr/logr v1.2.0
 	github.com/googleapis/gnostic => github.com/googleapis/gnostic v0.4.1
-	github.com/kubernetes-csi/csi-lib-utils => github.com/kubernetes-csi/csi-lib-utils v0.11.0
```

Comment:
It's better to keep this type of changes in a separate commit.

Reply:
Actually I ran go mod tidy after making my changes and changes in go.mod and go.sum were auto generated.
These changes were missed in a commit where this library was removed.